data.sql

```diff
@@ -115,6 +115,57 @@ UPDATE animals
 SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob')
 WHERE name IN ('Devimon','Plantmon');
 
+--Add Vets information
+
+INSERT INTO animals(ID,name,
+date_of_birth,escape_attempts,
+neutered,weight_kg) VALUES (9,
+'Agumon', '2020-02-03',0,true,10.23);
+
+INSERT INTO animals(ID,name,
+date_of_birth,escape_attempts,
+neutered,weight_kg) VALUES (10,
+'Devimon', '2017-05-12',5,true,11);
+
+--Add specializations Values
+
+INSERT INTO specializations(id,vet_id,spcs_id)
+VALUES (1,1,1),
+        (2,2,1),
+        (3,4,2);
```

Comment:
- [ ] Good job adding the queries but please kindly include a query to insert into the specializations table where vet name is `Stephanie Mendez` and the specie name is `Digimon`

Reply:
hi, but the question is How many different animals did Stephanie Mendez see not the type of the animals, could you please explain to me, i appreciate it

Reply:
ohhh i see, thank you im going to fix it